src/testUtils/awala/parcel.ts

```diff
@@ -1,19 +1,50 @@
+import { randomUUID } from 'node:crypto';
+
 import {
   type Certificate,
+  CertificationPath,
   issueGatewayCertificate,
   Parcel,
   type Recipient,
+  ServiceMessage,
+  SessionEnvelopedData,
+  type SessionKey,
   SessionlessEnvelopedData,
 } from '@relaycorp/relaynet-core';
-import type { NodeKeyPairSet } from '@relaycorp/relaynet-testing';
+import type { NodeKeyPairSet, PDACertPath } from '@relaycorp/relaynet-testing';
 import { addDays } from 'date-fns';
 
 import { bufferToArrayBuffer } from '../../utilities/buffer.js';
 
+import { ENDPOINT_ADDRESS } from './stubs.js';
+
 interface GeneratedParcel {
   readonly parcelSerialized: Buffer;
   readonly parcel: Parcel;
 }
+
+function serializeMessage(pdaPath: CertificationPath, endpointInternetAddress: string): Buffer {
+  const message = {
+    id: randomUUID(),
+    // eslint-disable-next-line @typescript-eslint/naming-convention, camelcase
+    pda_path: Buffer.from(pdaPath.serialize()).toString('base64'),
+    // eslint-disable-next-line @typescript-eslint/naming-convention, camelcase
+    endpoint_internet_address: endpointInternetAddress,
+  };
+  return Buffer.from(JSON.stringify(message));
+}
```

Comment:
This function adds too much complexity and is unnecessary in this repo. We're not processing pings here.

We should just create a stub `SERVICE_MESSAGE_CONTENT` (`Buffer`) and use it everywhere.

Reply:
The thing is I didn't understand the structure of the message. After working on the PDA pr I got good sense of it. Reworked this part